Cargo.toml

```diff
@@ -14,10 +14,11 @@ edition = "2021"
 thiserror = "1.0.40"
 chrono = "0.4.24"
 libc = "0.2.141"
-neli = "0.6.4"
 bitflags = "2.1.0"
 log = "0.4.17"
 
+neli = { git = "https://github.com/jbaublitz/neli.git", rev = "c6a18d78e33fbc7764c73cd19b165438dc53fe80" }
```

Comment:
Is there not a specific neli release that works here? I saw there was a [neli-v0.7.0-rc2](https://github.com/jbaublitz/neli/releases/tag/neli-v0.7.0-rc2) -- I'm guessing these changes require the updated neli, yeah? 

I'm perfectly ok merging if we can get on a non-rc release of neli.